src/lightning/fabric/plugins/precision/precision.py

```diff
@@ -42,7 +42,15 @@ def convert_module(self, module: Module) -> Module:
         """
         return module
 
-    @contextlib.contextmanager
+    @contextmanager
```

Comment:
this gets used by the "true" precision settings: 32-true, 64-true, and 16/bf16-true introduced in https://github.com/Lightning-AI/lightning/pull/17287

Reply:
Can you mention this feature in the CHANGELOG?